src/main/scala/rocketchip/BaseTop.scala

```diff
@@ -67,7 +67,8 @@ abstract class BaseTopModule[+L <: BaseTop, +B <: BaseTopBundle](
   val io: B = b
 
   val coreplex = p(BuildCoreplex)(outer.c, p)
-  val coreplexIO = coreplex.io
+  val coreplexIO = Wire(new BaseCoreplexBundle(outer.c)(p){})
+  val baseCoreplexIO = coreplex.io
```

Comment:
I decided to just avoid this problem entirely by using coreplex.io directly in the DirectConnection trait.